modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/service/impl/PLOEntryLocalServiceImpl.java

```diff
@@ -222,8 +229,25 @@ private void _validate(String key, String languageId, String value)
 			throw new PLOEntryKeyException.MustBeShorter(keyMaxLength);
 		}
 
-		if (!ArrayUtil.contains(PropsValues.LOCALES, languageId)) {
-			throw new PLOEntryLanguageIdException.MustBeAvailable(languageId);
+		Set<Locale> availableLocales = _language.getAvailableLocales();
+
+		String[] availableLanguageIds = new String[availableLocales.size()];
+
+		Iterator<Locale> iterator = availableLocales.iterator();
+
+		int i = 0;
+
+		while (iterator.hasNext()) {
```

Comment:
An `iterator` is most useful if you need to remove items from the collection being iterated. But in this case, you really are just "mapping" from one collection to another, right?

If so, then using `TransformUtil#unsafeTransform` will make this a little clearer.

Reply:
Done, thanks for the tip